For_university/experiment1.py

```diff
@@ -0,0 +1,72 @@
+import random
+
+SIZE_OF_UNIVERSE = 100
+NUMBER_OF_SUBSETS = 40
+CARDINALITY = 4
+REMAINING_SUBSET = NUMBER_OF_SUBSETS - CARDINALITY
+ELEMENTS_OF_UNIVERSE = list(range(SIZE_OF_UNIVERSE))
```

Comment:
you don't have to turn everything into a `list`. you can iterate directly any [sequence](https://docs.python.org/3/library/stdtypes.html#sequence-types-list-tuple-range)